### PR TITLE
disks: document the 'Bus' attribute for disks

### PR DESCRIPTION
**Pull request by fromanirh**

### CODE REVIEW

File: disks-and-volumes.md [hunk 1]
```diff
@@ -21,6 +21,10 @@ A disk can be made accessible via four different types:
 All possible configuration options are available in the
 [Disk API Reference](https://kubevirt.github.io/api-reference/master/definitions.html#_v1_disk).
 
+All types but **floppy** allow you to specify the `Bus` attribute. The `Bus` attribute determine
+how the disk will be presented to the Guest Operating System. **floppy** disks don't support
+the `Bus` attribute: they are always attached to the `fdc` bus.
```
Comment (fabiand) [inline disks-and-volumes.md]: Is fdc a bus or a device name?

I suppose that bus would be IDE, not?

File: disks-and-volumes.md [hunk 2]
```diff
@@ -73,7 +77,10 @@ spec:
       - name: mypvcdisk
         volumeName: mypvc
         # This makes it a disk
-        disk: {}
```
Comment (fabiand) [inline disks-and-volumes.md]: Is there still an example with an empty disk section?

Would be good to keep this example.

File: disks-and-volumes.md [hunk 3]
```diff
@@ -21,6 +21,10 @@ A disk can be made accessible via four different types:
 All possible configuration options are available in the
 [Disk API Reference](https://kubevirt.github.io/api-reference/master/definitions.html#_v1_disk).
 
+All types but **floppy** allow you to specify the `Bus` attribute. The `Bus` attribute determine
```
Comment (rmohr) [inline disks-and-volumes.md]: could you use `bus` instead of `Bus`? The user works with json/yaml and not with the struct names.

Comment (rmohr) [reply disks-and-volumes.md]: s/determine/determines/

File: disks-and-volumes.md (hunk 2 above)
Comment (rmohr) [reply disks-and-volumes.md]: +1

File: disks-and-volumes.md (hunk 1 above)
Comment (fromanirh) [reply disks-and-volumes.md]: fdc is a bus name (related doc: https://libvirt.org/formatdomain.html#elementsControllers)

floppies have their own bus and controller, AFAIK different from IDE. Se also https://en.wikipedia.org/wiki/Floppy-disk_controller and c

Comment (fabiand) [reply disks-and-volumes.md]: Wow - Thank you, I was never aware of this!

**Comment by fabiand** [comment]
+1 lgtm!

**Pull request merged**